website/content/en/v0.10.0/tasks/pod-density.md

```diff
@@ -24,7 +24,7 @@ The number of pods on a node is limited by the number of networking interfaces (
 
 [AWS VPC CNI v1.9 introduced prefix assignment.](https://aws.amazon.com/blogs/containers/amazon-vpc-cni-increases-pods-per-node-limits/) In short, a single ENI can provide IP addresses for multiple pods. Much higher pod densities are now supported. 
 
-Run the Karpenter controller with the enviornment variable `AWS_ENI_LIMITED_POD_DENSITY` (or the argument  `--aws-eni-limited-pod-density=true`) to enable nodes with more than 110 pods. 
+Run the Karpenter controller with the enviornment variable `AWS_ENI_LIMITED_POD_DENSITY: "false"` (or the argument  `--aws-eni-limited-pod-density=false`) to disable ENI-based pod density and allow nodes to host up to 110 pods. 
```

Comment:
Nice catch! The polarity of this boolean is backwards (defaults should be false). Let's fix in #1327

Reply:
Does this mean I dont have to do this ?
https://github.com/aws/karpenter/issues/2162

Reply:
Not sure what you mean. This value currently defaults to true.

Reply:
@FernandoMiguel correct, after https://github.com/aws/karpenter/issues/1327#issuecomment-1195605857 is applied ENI-based pod density will be disabled by default. Therefore, you will not need to set the environment variable to false anymore. However, the parsing issue you're experiencing with that helm module will still be there unless it gets fixed as well

Reply:
thanks @offzale